Makefile

```diff
@@ -24,6 +25,10 @@ install: man
 
 clean:
 	rm -f oci-runtime-tool runtimetest *.1
+	rm -f $(RUNTIME_TOOLS_LINK)
+
+$(RUNTIME_TOOLS_LINK):
+	ln -sfn $(CURDIR) $(RUNTIME_TOOLS_LINK)
```

Comment:
`-n` is [not in POSIX][1].  Do we really need it to support this use-case?

[1]: http://pubs.opengroup.org/onlinepubs/9699919799/utilities/ln.html